.eslintrc.js

```diff
@@ -91,10 +91,7 @@ module.exports = {
     'no-unused-vars': OFF,
     'no-nested-ternary': WARNING,
     '@typescript-eslint/no-unused-vars': [ERROR, {argsIgnorePattern: '^_'}],
-    '@typescript-eslint/ban-ts-comment': [
-      ERROR,
-      {'ts-expect-error': 'allow-with-description'},
-    ],
+    '@typescript-eslint/ban-ts-comment': OFF,
```

Comment:
This is going to be needed for webpack-merge, since it still uses @types/webpack

Reply:
you should update `webpack-merge` to `5.x`, and remove `@types/webpack-merge` as new version of package contains type definition

Reply:
That would be a breaking change for plugins that depend on it

Reply:
isn't that the same for just upgrading webpack to 5.x from 4.x? some of plugins can/will stop working

Reply:
No, Docusaurus plugins.

Reply:
`mergeStrategy` doesn't do anything anymore in webpack-merge 5, you have to manually program rules in order to customize merging, I don't think that its worth updating for now.

Reply:
Alright, I've updated webpack-merge, everything works just fine except for one test which I don't have the mental energy to rewrite

Reply:
was able to re-enable this rule without upgrading webpack-merge.

Will see if I can upgrade webpack-merge and keep retrocompatibility now